app/javascript/pulfalight/toc.es6

```diff
@@ -22,6 +27,10 @@ export default class TocBuilder {
       url = `${url}&expanded=true`
     }
 
+    if (this.getOnlineToggleValue()) {
+      url = `${url}&online_content=true`
```

Comment:
Do we need the URL query string here? The cookie should take care of that, right? Unless we want to be able to set the toggle via URL, but I don't see where that value would carry through.

Reply:
This is the url for the AJAX request. Not the url the user sees in the browser. It is needed.